ui-tests/cypress/support/pageObjects/content/types/AddPage.js

```diff
@@ -0,0 +1,47 @@
+import {TEST_ID_KEY, htmlElements, WebElement} from '../../WebElement';
+import Content from '../../app/Content.js';
+import AppPage from '../../app/AppPage.js';
+import TypesPage from './TypesPage.js';
+import EditPage from './EditPage';
+
+export default class AddPage extends Content {
+
+  nameInput = `${htmlElements.input}[name=name]`;
+  codeInput = `${htmlElements.input}[name=code]`;
+
+  getNameInput() {
+    return this.getContents()
+               .find(this.nameInput);
+  }
+
+  getCodeInput() {
+    return this.getContents()
+               .find(this.codeInput);
+  }
+
+  getSaveButton() {
+    return this.getContents()
+               .find(htmlElements.button)
+               .contains('Save');
+  }
+
+  typeName(value) {
+    this.getNameInput().type(value);
+  }
+
+  typeCode(value) {
+    this.getCodeInput().type(value);
+  }
+
+  save() {
+    this.getSaveButton().click();
+    cy.wait(1000);
```

Comment:
Please, add a _TODO_ comment on the wait to remind us we need to find a way to avoid explicit wait